Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -628,25 +628,30 @@ def genS(fragment,howMuch):
 U2000by1={'--relval': '2000,1'}
 U80by1={'--relval': '80,1'}
 
-hiAlca = {'--conditions':'auto:run2_mc_hi', '--era':'Run2_2016,Run2_HI'}
 hiAlca2011 = {'--conditions':'auto:run1_mc_hi'}
+hiAlca2015 = {'--conditions':'auto:run2_mc_hi', '--era':'Run2_2016,Run2_HI'}
+hiAlca2018 = {'--conditions':'auto:phase1_2017_realistic', '--era':'Run2_2017'}
```

Comment:
@davidlange6 Indeed this was brought up, and maybe we should.  Since we're currently reviving the HI reco, my thinking was better to use the stable 2017 geometry for the moment, and switch to 2018 detector at a later point.  I don't have a good feeling for how stable the 2018 geometry is.    Would your advice be to switch to the 2018 detector now?